lightly/models/modules/masked_autoencoder_timm.py

```diff
@@ -60,6 +62,7 @@ def __init__(
         proj_drop_rate: float = 0.0,
         attn_drop_rate: float = 0.0,
         norm_layer: Callable[..., nn.Module] = partial(LayerNorm, eps=1e-6),
+        initialize_weights: bool = False,
```

Comment:
```suggestion
        initialize_weights: bool = True,
```
I think this should be True by default, otherwise the existing tutorials/examples will break as weights will no longer be correctly initialized.